### ISSUE TITLE
Allow data:tree:import to query for existing records

### ISSUE THREAD

**Issue by neocolmartin** [forcedotcom/cli#710]
**Is your feature request related to a problem? Please describe.**
I can't import data using force:data:tree:import where a required lookup field needs to point to a record that already exists in the org. The specific example that I'm currently facing is that I want to import pricebook entries - the first pricebook entry for each product must be on the standard pricebook, which already exists.

**What are you trying to do**
When importing data using force:data:tree:import I would like to populate lookup fields with IDs from existing records from the target org. I'm happy to manually modify the JSON files & plans file, but I want to do this once and have it work for any org that I load to (e.g. to scratch orgs, where I may be loading it to many orgs over time). In particular, I want to be able to look up the standard pricebook ID to import price book entries.

**Describe the solution you'd like**
The solution provided here looks good, but it requires a change to the plug-in JS which means it's not portable between different devs: https://salesforce.stackexchange.com/a/241848/86924
In short, it allows for a `query` property on individual record entries in the JSON, which looks up an ID instead of inserting a record.

**Describe alternatives you've considered**
Scripting around the import to look up the ID and then modify the import files, but I don't like the idea of modifying the data import files every time, or having to create a copy of them.

**Comment by WillieRuemmele** [comment]
Hi @neocolmartin  check out the [announcement](https://github.com/forcedotcom/cli/issues/907) for the new [data](https://github.com/forcedotcom/cli/issues/907) plugin

**Comment by mshanemc** [comment]
I think this is fixed in the new data plugin released 4/1.  If not, re-open or create a new case.
